packages/radix-vue/src/Dialog/DialogClose.vue

```diff
@@ -1,31 +1,28 @@
-<script lang="ts">
-export interface DialogCloseProps {
-  asChild?: boolean;
-}
-</script>
-
 <script setup lang="ts">
 import { inject } from "vue";
-import { PrimitiveButton } from "../Primitive";
+import { PrimitiveButton, type PrimitiveProps } from "@/Primitive";
 import {
   DIALOG_INJECTION_KEY,
   type DialogProvideValue,
 } from "./DialogRoot.vue";
 
-const props = withDefaults(defineProps<DialogCloseProps>(), {
-  asChild: false,
-});
+export interface DialogCloseProps extends PrimitiveProps {}
+const props = defineProps<DialogCloseProps>();
 
 const injectedValue = inject<DialogProvideValue>(DIALOG_INJECTION_KEY);
+
+function handleClick() {
+  injectedValue?.closeModal();
+  injectedValue?.triggerButton.value?.focus();
+}
```

Comment:
this function does not respect `isCloseAutoFocus`, and would focus on triggerButton even tho when `isCloseAutoFocus="false"`.

maybe we need to wrap DialogContent's `closeModal`  with the triggerButton focus together

Reply:
minor bug here @khairulhaaziq . Else all lgtm!

Reply:
I did thought about this. however I thought just wanna leave it. I think I will change it, thanks!